Remove deprecated final_structure from phonopy example

pmrv opened the pull request:
`final_structure()` is deprecated, so I replaced it (on github directly, so I didn't run it again, but should be fine).

jan-janssen commented on pyiron_atomistics/structure/ovito.py:
```diff
@@ -36,4 +36,16 @@ def analyse_ovito_cna_adaptive(atoms, mode='total'):
+def analyse_ovito_centro_symmetry(atoms, num_neighbors=12):
+    """
+    Args:
+        mode (str): ['total', 'numeric', 'str']
```
Can you update the DocString to match the parameters?

pmrv committed:
Remove deprecated final_structure from phonopy example

`final_structure()` is deprecated, so I replaced it (on github directly, so I didn't run it again, but should be fine).

jan-janssen reviewed (approved):
Looks good to me
